services/painting-surface-defect-detection-model-service/tests/test_model_loader.py

```
@@ -0,0 +1,370 @@
+            'model', 'model_config'
+        ]
+        
+        for attr in required_attributes:
+            assert hasattr(self.model_loader, attr)
+
+    def test_defect_classes_structure(self):
+        """결함 클래스가 올바른 구조를 가지고 있는지 테스트"""
+        classes = self.model_loader.defect_classes
+        
+        # 모든 키가 정수인지 확인
+        assert all(isinstance(k, int) for k in classes.keys())
```

Ruff (0.12.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)